### ISSUE TITLE
CSV Problem during Import

### ISSUE THREAD

**Issue by obstschale** [RubenGees/Vocabletrainer#5]
I converted a PDF to CSV and I encountered a problem with your CSV import. Your app does not consider text delimiter in CSVs. For example I have a vocabulary with an comma in it. In CSV you can use " (double quote) or ' (quote) to mark a string as one entry. If you consider this your app could also import this correctly:

```
sancocho (LA),"Suppe mit Gemüse, Kochbanane und Fleisch oder Fisch",Spanisch-Deutsch
```

**Comment by rubengees** [comment]
I didn't even knew that there is some kind of "official" csv. I always thought it was application dependent  how it is used. Do you have a link or something where I can found more info about that?
But yes, I can implement that.

**Comment by obstschale** [comment]
Yes there is a RFC which explains the correct use of CSV. [RFC 4180](https://www.rfc-editor.org/rfc/rfc4180.txt). In section 2 there are examples how to use double quotes :)

**Comment by rubengees** [comment]
Okay, thank you, will have a look.
